### DISCUSSION TITLE
Layout Ideen

### DISCUSSION THREAD

**Discussion by Zefau** [Zefau/ioBroker.jarvis#928] [Show and tell]
_Originally posted by @RkcCorian in https://github.com/Zefau/ioBroker.jarvis/issues/805#issuecomment-910597335_

> @WW1983 : Wie hast Du das mit dem Hintergrundbild gemacht, sowie Luftfeuchtigkeitskachel, etc. Sieht echt geil aus. Da dies hier aber vermutlich nicht der richtige Ort ist, ich aber nicht weiß wie ich Dich sonst und wo kontaktieren kann wäre ich auch für die Info dankbar 🙈
@Zefau : Saugeil was man mit JARVIS alles machen kann, vielen Dank! Hab bereits gespendet 😁

**Comment by Zefau** [comment]
@RkcCorian kennst du https://forum.iobroker.net/topic/37661/showcase-jarvis-just-another-remarkable-vis ?

**Comment by RkcCorian** [comment]
@Zefau : Vielen Dank! Ich schaue später mal detaillierter drauf. Ich habe es aber gerade zumindest mal überflogen. Ich hatte schon in der Vergangenheit mit CSS experimentiert (Blinken), leider hatte es nie einen Effekt. Ich habe gerade mal in den CSS-Bereich
```
body {
  background-color: #0cbaba;
  background-image: linear-gradient(315deg, #0cbaba 0%, #380036 74%);
}
 
.jarvis-AppBar {background: none}
 
.jarvis-Toolbar, .jarvis-Tabs, .jarvis-Widget {
  background-color: rgba(255,255,255,0.2)
}
 
.jarvis-Tabs {
  border-bottom: 0;
}
```
reinkopiert. Es bewirkt bei mir aber nichts. Woran kann das liegen?

**Reply by Zefau** [reply]
Wenn du v3 laufen hast dann gibt es die CSS Klassen nicht mehr. Die sind nun komplett anders, da v3 komplett neu ist.

**Reply by RkcCorian** [reply]
@Zefau : Hatte es in v2 und v3 probiert. Da ich schnellstmöglich (sobald ein paar für mich relevante Bugs behoben sind) umsteigen möchte brauche ich nur die Info für v3. Wo kann ich diese finden?

**Reply by Zefau** [reply]
Aktuell nur das v2 Wiki. Das v3 Wiki steht noch aus. Welche Bugs sind für dich im Fokus?

**Comment by RkcCorian** [comment]
@Zefau : Ich glaube es fehlt (zumindest für mich nicht mehr soo viel), vgl. unten. Was ich bislang nicht weiß ob ich noch Skalierungsfehler auf verschiedenen Bildschirmen bekommen wenn ich feintunen würde und es dann "exakt" passen soll. Das würde ich erst später merken :-)

**Benutzte OS & Browser**

- Windows 10 (21H1) mit MS Edge (92.0.902.78 (64-Bit)) _und selten Mozilla Firefox (91.0.2 (64-Bit))_
- Android 10 mit Hermit Browser (19.5.5) _und manchmal MS Edge (46.06.4.5161)_
- Android 11 mit Fully Browser (1.43.6)

**Aktuell bekannte Blocker**

- #926
- #688

**Darüber hinaus nicht gut aber kann ich eine Zeit lang mit leben**

- DisplayImage-Widget: Keine Möglichkeit das Bild (bei mir QR-Code) mittig anzuzeigen (Kosmetik)
- _Das Menüband ganz oben (Standardkonfig) ist für mich etwas zu klein, die Größe sollte konfigurierbar sein. Hintergrund: Der Fully Browser läuft bei mir auf einem Tablet an der Wand mit Touch-Bedienung. Es ist tlw. nicht gut mit den Fingern bedienbar_ => Nicht mehr ganz so wichtig, habe erst jetzt den Dense Mode entdeckt, welchen ich hierfür auch nutzen könnte. Der hat allerdings noch einen Bug, welcher hierfür behoben werden müsste, vgl. #805 

**Weiteres**

- Invertierung bei boolean-Werten, vgl. unsere Diskussion innerhalb #926

**Comment by RkcCorian** [comment]
@Zefau Nach den neusten BugFixes habe ich mich (und damit meine Familie) mal auf die v3 gewagt. Soweit scheint das Meiste zu gehen, ich werde noch ein paar Issues erstellen...

Bzgl. dem Hintergrundbild (ursprüngliches Thema oben)... hast Du einen Tipp, wie das mit der v3 geht oder verweist Du auf später zu erstellende Doku? :-)

Viele Grüße!

**Comment by WW1983** [comment]
Sorry @RkcCorian habe erst zu spät gesehen, dass du eine Frage an mich hattest. Habe JARVIS aktuell deaktiviert und deshalb hier wenig  geschaut. Wenn das Thema mit dem Hintergrundbild noch aktuell ist. Das Hintergrundbild habe ich über CSS mit der folgenden Zeile eingefügt: 

.q-layout { background-image: url("https://wallpapercave.com/wp/wp2914444.jpg")}

Du kannst mich aber auch im Forum anschreiben: https://forum.iobroker.net/user/ww1983

**Reply by Zefau** [reply]
Oh, wieso deaktiviert? :-o

**Reply by WW1983** [reply]
Hat nichts mit JARVIS zu tun. Habe gesamt IoBroker deaktiviert und teste gerade Home Assistant

**Reply by RkcCorian** [reply]
@WW1983 Vielen Dank! Ich versuche Dir direkt zu schreiben. Bin neu in dieser "Welt" und habe mir schon einiges angeeignet.
Ggf. würde mich Dein Vergleich zu Home Assistant interessieren. Ich bin aktuell sehr zufrieden, insb. seit Admin5 und werde aktuell in keinem Fall wechseln aber ich schaue auch gerne mal über den Tellerrand hinaus.
